### PR TITLE
add replication_delay to namespace metrics

### PR DESCRIPTION
**Pull request by rdhabalia**
### Motivation

Right now, broker metrics provide `replicationDelay` per topic level but we have setup monitoring on namespace metrics `./pulsar-admin  broker-stats monitoring-metrics` and we also need `replicationDelay` at namespace level to get estimate delay from namespace graphs.

### Modifications

Find out max replication-delay among all the topics of a namespace and use it to namespace-metrics to provide estimate repl-dealy for a namespace.

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/NamespaceStats.java [hunk 1]
```diff
@@ -72,6 +74,7 @@ public Metrics add(String namespace) {
         dMetrics.put("brk_no_of_consumers", consumerCount);
         dMetrics.put("brk_msg_backlog", msgBacklog);
         dMetrics.put("brk_replication_backlog", msgReplBacklog);
+        dMetrics.put("brk_replication_delay_second", msgReplDelayInSeconds);
```
Comment (merlimat) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/NamespaceStats.java]: Should it be made explicit this is the max value? That would help people building aggregations in dashboard as well.

**Comment by rdhabalia** [comment]
@merlimat addressed it.

**Comment by rdhabalia** [comment]
rerun integration test

**Comment by rdhabalia** [comment]
run integration tests

**Comment by jiazhai** [comment]
run integration tests

**Comment by rdhabalia** [comment]
run integration tests

**Comment by sijie** [comment]
run integration tests

**Comment by sijie** [comment]
run integration tests

**Pull request merged**